### ISSUE TITLE
Support for self-signed certificates in the setWebhook 

### ISSUE THREAD

**Issue by GustavoRPS** [yagop/node-telegram-bot-api#23]
On August 29, 2015 Telegram Bot API added support for self-signed certificates: upload your certificate using the certificate parameter in the setWebhook method.

My sugestion for implementation:

```
/**
 * Specify an url to receive incoming updates via an outgoing webHook.
 * @param {String} url URL where Telegram will make HTTP Post. Leave empty to
 * delete webHook.
 * @see https://core.telegram.org/bots/api#setwebhook
 */
TelegramBot.prototype.setWebHook = function (url) {
  var path = 'setWebHook';
  var qs = {url: url};

  if (this._WebHook)
    if (this._WebHook._webServer.cert)
      qs.formData = {
        certificate: this._WebHook._webServer.cert
      }
  
  return this._request(path, {qs: qs})
    .then(function (resp) {
      if (!resp) {
        throw new Error(resp);
      }
      return resp;
    });
};
```

**Comment by periloso** [comment]
+1

**Comment by yagop** [comment]
Done on bb6a0bb3ff2841c0f844050db7a6125a311a58fd. Released on v0.14.0
I prefer to set it as setWebHook param, some people run webHook on HTTP mode behind HTTPS proxy (like ngnix)
